backend/api/system/general.py

```diff
@@ -227,3 +235,40 @@ def post(self):
                 "Error": "Unable to fetch application keys",
                 "SubCode": "InternalServerError",
             }, 500
+
+
+class SystemReleaseAPI(Resource):
+    def post(self):
+        """
+        Fetch latest release version form github and save to database.
+        ---
+        tags:
+          - system
+        produces:
+          - application/json
+        responses:
+          201:
+            description: Saved version successfully to database
+          502:
+            description: Couldn't fetch latest release from github
+          500:
+            description: Internal server error
```

Comment:
Can we authenticate/authorize only certain roles or people/bots with keys to call this API and return 403 forbidden otherwise?

Reply:
We can make this endpoint restricted to sys admin role only, but for that we need to generate token with no expiry or longer expiry date as our session token expires every 7 days. Since people can't abuse this request i think we can leave it open to all users.